sphinx_ncs_theme/layout.html

```diff
@@ -13,8 +13,32 @@
   rel="shortcut icon"
   href="{{ pathto('_static/images/favicon.ico', 1) }}"
 />
+{%- if theme_set_default_announcement|tobool %}
+<style>
+  .announcement {
+    display: block;
+  }
+</style>
```

Comment:
shouldn't this be in nordic.css?

Reply:
It seems to me that it is not possible to inject jinja template code into css files. At least it does not work for me with the `{%- ... %}` syntax. Is there a different syntax for css files?

Reply:
Seems to work